src/pages/CreateRoute/RouteManagement/api/getRoutesData.ts

```diff
@@ -0,0 +1,81 @@
+import { Driver, Order } from '@/interfaces/interfaces';
+import { setOrdersToDrivers } from '@/store/slices/ordersToDriversSlice';
+import { store } from '@/store/store';
+import axios, { AxiosError } from 'axios';
+import { toast } from 'react-toastify';
+
+interface Assignment {
+  driver: Driver;
+  orders: Order[];
+}
+
+const assignOrdersToDrivers = (
```

Comment:
why you do it on fe? we will not have all orders on fe and all drivers. I would move any logic to be, frontend mostly to display data only

Reply:
I am performing a GET request for the selected orders and drivers and then building approximate routes based on them. I did this to make it easier to catch errors in the future when order times overlap (I think this way it will be simpler). Additionally, I believe this is not significant for the client, and it reduces the load on the server

Reply:
server we can increase as match as we can. but work computers usually not powerful, always heavy logic on be. you have all needed data on be, move calculation there

Reply:
I want to create a separate task where I will both redesign the order allocation algorithm for drivers and handle the error that occurs when multiple orders have the same time

Reply:
ok

Reply:
next time attach in the comment new tiket, so I'm sure that you will not forget

Reply:
ok